ironfish/src/memPool/memPool.ts

```diff
@@ -123,6 +124,14 @@ export class MemPool {
     const hash = transaction.hash().toString('hex')
     const sequence = transaction.expirationSequence()
 
+    if (
+      getTransactionSize(transaction.serialize()) >
+      this.chain.consensus.MAX_BLOCK_SIZE_BYTES - getBlockHeaderSize() - 562 - 2
```

Comment:
What are these 2 numbers?  It would be nice to write a comment here, this is gonna be hard for someone else to figure out. In that case I'd prefer if this was just a variable as MAX_TRANSACTION_SIZE_BYTES somewhere that had documentation on why its what it is.

This is trying to encode "remaining size after a block header", but that won't work if the block header ever becomes variable size.